src/plugins/vis_type_vega/public/data_model/vega_parser.ts

```diff
@@ -448,18 +495,18 @@ export class VegaParser {
    * @private
    */
   _parseSchema() {
-    if (!this.spec.$schema) {
```

Comment:
if we put a `if (!this.spec) { return false; }` on the top of the function, the assertion operators below aren't necessary anymore.